src/components/WhaleSightingViewer/WhaleSightingFilters.tsx

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import {
+    ChangeEvent,
+    useState,
+    useEffect,
+    FormEvent
+} from "react";
+import { fetchSpeciesQuery, TailType, Size, SpeciesSearch, WhaleSpecies } from "../../clients/apiClient";
+import "./WhaleSightingFilters.scss";
+
+interface WhaleSightingDropdownProps {
+    selectedWhaleSpecies: string;
+    setSelectedWhaleSpecies: React.Dispatch<React.SetStateAction<string>>;
+    selectedColour: string;
+    setSelectedColour: React.Dispatch<React.SetStateAction<string>>;
+    selectedTailType: string;
+    setSelectedTailType: React.Dispatch<React.SetStateAction<string>>;
+    selectedSize: string;
+    setSelectedSize: React.Dispatch<React.SetStateAction<string>>;
+    setMinLat: React.Dispatch<React.SetStateAction<string>>,
+    setMaxLat: React.Dispatch<React.SetStateAction<string>>,
+    setMinLog: React.Dispatch<React.SetStateAction<string>>,
+    setMaxLog: React.Dispatch<React.SetStateAction<string>>,
+    handleSearch: (event: FormEvent<HTMLFormElement>) => void,
+}
+
+export function WhaleSightingFilters(
+    {
+        selectedWhaleSpecies, setSelectedWhaleSpecies,
+        selectedColour, setSelectedColour,
+        selectedTailType, setSelectedTailType,
+        selectedSize, setSelectedSize,
+        setMinLat, setMaxLat,
+        setMinLog, setMaxLog,
+        handleSearch
+    }: WhaleSightingDropdownProps) {
+
+
+    const [listWhaleSpecies, setListWhaleSpecies] = useState<WhaleSpecies[]>([]);
+    let search: SpeciesSearch =
+    {
+        tailType: null,
+        size: null,
+        colour: null
+    };
+
+    useEffect(() => {
+        fetchSpeciesQuery(search)
+            .then(response => setListWhaleSpecies(response));
+    }, []);
+
+    const handleChangeSpecies = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedWhaleSpecies(event.target.value);
+    };
+
+    const handleChangeColour = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedColour(event.target.value);
+    };
+
+    const handleChangeTailType = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedTailType(event.target.value);
+    };
+
+    const handleChangeSize = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedSize(event.target.value);
+    };
+
+    let colourList = [...new Set(listWhaleSpecies.map(ws => ws.colour))];
+
+    return (
+        <div className="filterCard">
+            <form className="whalesightingfilterform" onSubmit={(event) => handleSearch(event)}>
+                <div className="filter-grid">
+                    <label htmlFor="whalespecies">
+                        Whale Species
+                        <br></br>
+                        <select className="selectOption" value={selectedWhaleSpecies} onChange={handleChangeSpecies}>
```

Comment:
As the `handle..` functions are a single line we can put them directly into the `onChange` function
```suggestion
                        <select 
                            className="selectOption" 
                            value={selectedWhaleSpecies} 
                            onChange={e => setSelectedWhaleSpecies(e.target.value)}>
```